<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1009</width>
    <height>679</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Monitor de Processos</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>981</width>
      <height>351</height>
     </rect>
    </property>
    <property name="sortingEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCustomPlot" name="plot" native="true">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>380</y>
      <width>381</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>370</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Gerenciar Processos:</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>390</y>
      <width>141</width>
      <height>162</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="4" column="1">
      <widget class="QPushButton" name="pausarButton">
       <property name="text">
        <string>Pausar</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="pidText"/>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="continuarButton">
       <property name="text">
        <string>Continuar</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>PID:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QPushButton" name="encerrarButton">
       <property name="text">
        <string>Encerrar</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>390</y>
      <width>153</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="1">
      <widget class="QLineEdit" name="cpuText"/>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="trocarCpuButton">
       <property name="text">
        <string>Trocar CPU</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="pidCpuText"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>PID:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>CPU:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Filtrar:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="command"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>560</y>
      <width>381</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="2">
      <widget class="QLabel" name="label_7">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>CPU 2:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_8">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>CPU 3:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QProgressBar" name="cpuUsage_1">
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_9">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>CPU 4:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>CPU 1:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QProgressBar" name="cpuUsage_2">
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QProgressBar" name="cpuUsage_3">
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QProgressBar" name="cpuUsage_4">
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1009</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>command</sender>
   <signal>textEdited(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>filtrar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>95</x>
     <y>428</y>
    </hint>
    <hint type="destinationlabel">
     <x>97</x>
     <y>465</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pidText</sender>
   <signal>textEdited(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>validarPID()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>715</x>
     <y>445</y>
    </hint>
    <hint type="destinationlabel">
     <x>674</x>
     <y>566</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pausarButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>pausarProcesso()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>679</x>
     <y>522</y>
    </hint>
    <hint type="destinationlabel">
     <x>627</x>
     <y>545</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>continuarButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>continuarProcesso()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>699</x>
     <y>477</y>
    </hint>
    <hint type="destinationlabel">
     <x>621</x>
     <y>485</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pidCpuText</sender>
   <signal>textEdited(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>validarPidCpu()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>877</x>
     <y>449</y>
    </hint>
    <hint type="destinationlabel">
     <x>976</x>
     <y>416</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cpuText</sender>
   <signal>textEdited(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>validarPidCpu()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>903</x>
     <y>483</y>
    </hint>
    <hint type="destinationlabel">
     <x>970</x>
     <y>498</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>trocarCpuButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>trocarCpu()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>847</x>
     <y>524</y>
    </hint>
    <hint type="destinationlabel">
     <x>826</x>
     <y>559</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>encerrarButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>matarProcesso()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>735</x>
     <y>567</y>
    </hint>
    <hint type="destinationlabel">
     <x>699</x>
     <y>638</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>start()</slot>
  <slot>filtrar()</slot>
  <slot>validarPID()</slot>
  <slot>pausarProcesso()</slot>
  <slot>continuarProcesso()</slot>
  <slot>validarPidCpu()</slot>
  <slot>slot1()</slot>
  <slot>trocarCpu()</slot>
  <slot>matarProcesso()</slot>
 </slots>
</ui>
